help_function/src/onedpl_test_group_load.cpp

```diff
@@ -0,0 +1,114 @@
+// ====------ onedpl_test_group_load.cpp---------- -*- C++ -* ----===////
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//
+// ===----------------------------------------------------------------------===//
+#include <sycl/sycl.hpp>
+#include <dpct/dpct.hpp>
+#include <dpct/dpl_utils.hpp>
+#include <iostream>
+#include<oneapi/dpl/iterator>
+
+
+bool helper_validation_function(const int* ptr,const char* func_name){
+  // Used for validation of output and expected output sequences
+  oneapi::dpl::counting_iterator<int> expected(0);
+  
+  for (int i = 0; i < 512; ++i) {
+    if (ptr[i] != expected[i]) {
+      std::cout << func_name <<" failed\n";
+      std::ostream_iterator<int> Iter(std::cout, ", ");
+      std::copy(ptr, ptr + 512, Iter);
+      std::cout << std::endl;
+      return false;
+    }
+  }
+
+  std::cout << func_name <<" pass\n";
+  return true;
+}
+
+bool test_load_blocked() {

```

Comment:
My recommendation is that we combine `test_load_blocked` and `test_load_striped` and have a template parameter to dictate the strategy (`BLOCK_LOAD_DIRECT` or `BLOCK_LOAD_STRIPED`) since the functions are almost the same.